### ISSUE TITLE
[Bug] Upgrading from 3.0.1PE to 3.1.0PE missing custom Widget Bundles

### ISSUE THREAD

**Issue by imoon** [thingsboard/thingsboard#3304]
**Describe the bug**
After upgrading from 3.0.1PE to 3.1.0PE using this process (https://thingsboard.io/docs/user-guide/install/pe/upgrade-instructions/#upgrading-to-31pe) caused my custom widget bundle to disappear. Rebooting has not worked.

**Your Server Environment**

* AWS marketplace 

**Your Client Environment**

**Desktop (please complete the following information):**

* OS: MacOS
* Browser: Chrome
* Version: 84

**Your Device**

* Connectivity
  * HTTP

* Device vendor and model

**To Reproduce**

**Expected behavior**
After upgrading I expected to view custom widget bundles created in 3.0.1PE but they are not available in the Widget Library or on any of the dashboards.

**Comment by imoon** [comment]
Some troubleshooting results:

The widget bundle is in the database as are the widgets themselves. The widget bundle is even available in other areas of the UI such as the Select Widget pane when adding a new widget to a dashboard. The only place they seem to be unavailable is the Widget Bundle list page.

**Comment by vvlladd28** [comment]
Hi @imoon ,
Please check, if you have one or multiple pages with widget bundles?

**Comment by imoon** [comment]
@vvlladd28 Yes, that is the problem. There are now 16 widget bundles and they were sorted by date and paginated to the default 10 items. I guess after the update the built-in bundles were updated so they had a more recent date than the bundle I had created. I had grown accustomed to my bundles being at the top of the list.

For these lists, is it possible to change the default pagination to a number more than 10?
